### ISSUE TITLE
Determine xrfi compute requirements

### ISSUE THREAD

**Issue by adampbeardsley** [HERA-Team/hera_qm#74]
We've talked about four strategies for rfi flagging:
A) xrfi_simple run on all visibilities
B) xrfi (uses median filter), run on all vis (will be expensive)
C) xrfi run on subset of data
D) xrfi run on chisq

Basically, we need to know if B is feasible.

**Comment by plaplant** [comment]
Here is a benchmarking that I ran comparing omnical (currently the most expensive step in the pipeline by a fair margin) with option (B) above. For reference, option (A) takes about 30 seconds per polarization, for a total of 2 minutes per observation. I ran omnical in 4 polarization mode, and option (B) with the default settings. Also, each test was run 5 times to get an accurate sense of timing. This was performed on still1, so it's a fair test of how the hardware would actually perform.

[benchmark_170915.txt](https://github.com/HERA-Team/hera_qm/files/1308305/benchmark_170915.txt)

The tl;dr is that omnical takes ~20 minutes to run, and option (B) for all polarization takes ~28 minutes. There is more to the story though, since omnical runs all 4 polarizations as a single thread, and RTP parallelizes over different RFI operations (so in practice it would generally take less than 28 minutes to do all the RFI flagging).

Going off of file creation times for RTP products, a single night's worth of data takes ~9 hours to complete. (I can quantify this more if people would like, to make a more informed decision.) However, we can make this process take substantially less time, by increasing the computational load on each still. I think we're putting on average a <10% compute load on each of the stills, and there is equally little memory pressure. So we can make the walltime of a single night's processing smaller without too much effort.

All this to say, it seems like we have the budget to do option (B). In the coming week, I'll work on tweaking RTP settings to increase the load and make things take less time, but even projecting out to the end of the season (when things will likely take 2-4x as much compute time), we should still be able to keep RTP living up to its "real time" moniker.
